### DISCUSSION TITLE
Failing to boot

### DISCUSSION THREAD

**Discussion by sixilli** [regolith-linux/regolith-desktop#818] [Q&A]
For some reason after I ran an update yesterday I'm unable to login. This post covers what I've tried and what I'm experiencing. I've also tried uninstalling and reinstalling gdm. Any advise would be greatly appreciated. I'm not sure what else to try and any help would be greatly appreciated. 

https://askubuntu.com/questions/297080/ubuntu-boots-to-a-black-screen-with-blinking-a-underscore-character-after-releas

**Comment by kgilmer** [comment]
Hi @sixilli , are you able to see a terminal session by following this suggestion?  https://askubuntu.com/a/349410

I would look in `/var/log/syslog` for clues as to what's preventing your X11 system starting..
